src/AbstractDbEntityFetcher.php

```diff
@@ -91,6 +91,23 @@ protected function getFetchPaginationResult(array $objects, $pagination)
     }
 
     /**
+     * Helper method to get pagination result as objects if pagination is requested.
+     *
+     * @param array $rows
+     * @param bool  $pagination
+     * @return array
+     */
+    protected function getFetchPaginationAsDbEntitiesResult(array $rows, $pagination)
```

Comment:
Not too sure about removing `$pagination`. That would involve refactoring in our private repository. I'll refer you to the related diff. 🙂